tajo-rpc/src/main/java/org/apache/tajo/rpc/NettyClientBase.java

```diff
@@ -116,34 +124,24 @@ public void connect(InetSocketAddress addr) throws ConnectTimeoutException {
     handleConnectionInternally(addr);
   }
 
-  public boolean isConnected() {
-    return getChannel().isConnected();
+  public boolean isActive() {
+    return getChannel().isActive();
   }
 
   public InetSocketAddress getRemoteAddress() {
-    if (channelFuture == null || channelFuture.getChannel() == null) {
+    if (channelFuture == null || channelFuture.channel() == null) {
       return null;
     }
-    return (InetSocketAddress) channelFuture.getChannel().getRemoteAddress();
+    return (InetSocketAddress) channelFuture.channel().remoteAddress();
   }
 
   public Channel getChannel() {
-    return channelFuture.getChannel();
+    return channelFuture.channel();
   }
 
   @Override
   public void close() {
-    if(this.channelFuture != null && getChannel().isOpen()) {
-      try {
-        getChannel().close().awaitUninterruptibly();
```

Comment:
Sorry for that. I would like to implement reusable codes with netty channels, but I didn't have any idea on this and did not recover this code block yet. I will fix it soon.